trino/README.md

```diff
@@ -0,0 +1,60 @@
+# Agent Check: Trino
+
+## Overview
+
+This check collects [Trino][1] metrics, such as the following examples:
+
+- Overall activity metrics: completed/failed queries, data input/output size, execution time.
+- Performance metrics: cluster memory, input CPU, execution CPU time.
+
+## Setup
+
+### Installation
+
+The Trino check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
```

Comment:
Integrations in `integrations-extras` are not installed with the agent. Customers will manually need to download the Trino check. See https://github.com/DataDog/integrations-extras/tree/master/fluentbit#installation-1 for another extras-based check's installation instructions,

Reply:
Done!